scripts/publishing-common.sh

```diff
@@ -30,12 +30,18 @@ supported_os() {
 }
 
 dryval() {
-	if ${DRYRUN} ; then
+	case ${DRYRUN} in
```

Comment:
Though this improves the DRYRUN/DEBUG handling, this should happen as close to user input as possible. This could be moved into a function for true/false sanitization and used at the flag's parsing.

Example use case: https://github.com/aws/amazon-ecs-agent/blob/13a0fabe5230c677a5b75a231256581fab08d691/scripts/publish-staged#L100

Reply:
added "else DRYURN=true" statement.

Reply:
I like it!